concepts/05 UI Components/Chart/02 Overview/10 Key Features.md

```diff
@@ -0,0 +1,24 @@
+- **Series Options**    
+The Chart component features over 20 [types of series](/concepts/05%20UI%20Components/Chart/11%20Series%20Types/10%20Line%20Series.md '/Documentation/Guide/UI_Components/Chart/Series_Types/'). [Combine](/concepts/05%20UI%20Components/Chart/10%20Series/00%20Overview.md '/Documentation/Guide/UI_Components/Chart/Series/Overview/') them, [customize](/Documentation/Guide/UI_Components/Chart/Series/Customize_Appearance/) appearance, and configure individual [points](/concepts/05%20UI%20Components/Chart/14%20Series%20Points/00%20Overview.md '/Documentation/Guide/UI_Components/Chart/Series_Points/Overview/').
```

Comment:
I think we can consider adding all map of different chart types here (similar with [kendo](https://www.telerik.com/kendo-jquery-ui/charts or [MUI](https://mui.com/x/react-charts/#all-mui-x-charts-components)) 
we can use the style from all component list

Please take a look to the [comparison](https://devexpress.sharepoint.com/:x:/s/devextreme/EXyj0qNp6fxGiSm5vLsrkroBXF_J6j2AKspMlGNzpxp5nw?e=JkhWfv&nav=MTVfezNCMzUxODJDLUYyNzctNDk2OS1CRUI2LUIyODZGOTFDQzVDNH0 
) that we have